Packs/AccentureCTI/Integrations/ACTIVulnerabilityQuery/test_data/baseintegration-dummy.json

```diff
@@ -0,0 +1,3 @@
+{
+    "dummy": "this is a dummy response"
```

Comment:
you can remove this file if your mock data is already in `response_constants.py`.